frameworks/cassandra/tests/test_tls.py

```diff
@@ -0,0 +1,246 @@
+import json
+import os
+import textwrap
+import uuid
+from typing import List
+
+import dcos.http
+import pytest
+import requests
+import shakedown
+
+import sdk_cmd
+import sdk_install
+import sdk_jobs
+import sdk_networks
+import sdk_plan
+import sdk_security
+import sdk_utils
+
+
+from tests.config import (
+    DEFAULT_NODE_ADDRESS,
+    DEFAULT_NODE_PORT,
+    DEFAULT_TASK_COUNT,
+    PACKAGE_NAME,
+    _get_test_job
+)
+
+
+@pytest.fixture(scope='module')
+def dcos_ca_bundle():
+    """
+    Retrieve DC/OS CA bundle and returns the content.
+    """
+    url = shakedown.dcos_url_path('ca/dcos-ca.crt')
+    resp = dcos.http.request('get', url)
+    cert = resp.content.decode('ascii')
+    assert cert is not None
+    return cert
+
+
+@pytest.fixture(scope='module')
+def service_account():
+    """
+    Creates service account with `hello-world` name and yields the name.
+    """
+    name = PACKAGE_NAME
+    sdk_security.create_service_account(
+        service_account_name=name, service_account_secret=name)
+     # TODO(mh): Fine grained permissions needs to be addressed in DCOS-16475
+    sdk_cmd.run_cli(
+        "security org groups add_user superusers {name}".format(name=name))
+    yield name
+    sdk_security.delete_service_account(
+        service_account_name=name, service_account_secret=name)
+
+
+@pytest.fixture(scope='module')
+def cassandra_service_tls(service_account):
+    sdk_install.install(
+        PACKAGE_NAME,
+        DEFAULT_TASK_COUNT,
+        service_name=service_account,
+        additional_options={
+            "service": {
+                "service_account_secret": service_account,
+                "service_account": service_account,
```

Comment:
`service_account_secret` and `service_account` have the same name?

Reply:
Yes, I've used same name. I don't think it matters what names are we going to use here.